src/style.css

```diff
@@ -34,23 +41,30 @@
   width: 40%;
 }
 
+.refresh:hover {
+  background-color: #c5aa50;
+}
+
 #leaderboard {
   display: flex;
   flex-direction: column;
   grid-column: 2;
   grid-row: 2;
-  border: 2px solid black;
+
+  /* border: 2px solid black; */
   padding-left: 5px;
 }
 
 .score-heading {
   display: flex;
+  justify-content: center;
   align-items: center;
   grid-column: 4;
   grid-row: 1;
 }
 
 .add-score {
```

Comment:
- [ ] Nice design. However, kindly note we as much as possible want our design to mimic very closely the [wireframe](https://github.com/microverseinc/curriculum-javascript/blob/main/leaderboard/images/leaderboard_wireframe.png) design. Kindly ensure that your scores are appearing in a _database alternating colours_, as seen in the  [wireframe](https://github.com/microverseinc/curriculum-javascript/blob/main/leaderboard/images/leaderboard_wireframe.png)  template.


**Your design:**
![image](https://user-images.githubusercontent.com/12745474/174194763-e3b7050c-ed63-4481-8bb3-266598b8990d.png)



**Wireframe design:**
![image](https://user-images.githubusercontent.com/12745474/174194852-4b5626db-de30-4756-9e30-8ddd38e6e3a1.png)